index.html

```diff
@@ -3954,6 +3954,10 @@ <h2>Methods</h2>
                   <code>pt_table</code> conflicts described in Section 8.3 or other
                   reasons, <a>reject</a> <var>p</var> with <code>InvalidParameters</code>.</p>
                 </li>
+                <li>
+                  <p>If <code>transport</code> is not set, <a>reject</a> <var>p</var>
+                  with a newly created <code>TypeError</code>.</p>
```

Comment:
While TypeError is consistent with the usage in (send())[http://draft.ortc.org/#dom-rtcrtpsender-send] I wonder if it should be InvalidStateError in both places.